src/toil/leader.py

```diff
@@ -51,6 +52,17 @@ def __init__(self, jobStore):
         self._worker.start()
 
     @staticmethod
+    def writeLogs(jobStoreID, jobLogList):
+        def logFileName(jobStoreID):
```

Comment:
If it's only called once there is no need to extract a function. It just adds confusion.